content/02.intro.md

```diff
@@ -26,11 +26,11 @@ In particular, deep learning approaches can be used both in *supervised* applica
 Deep learning methods may in fact combine both of these steps.
 When sufficient data are available and labeled, these methods construct features tuned to a specific problem and combine those features into a predictor.
 In fact, if the dataset is "labeled" with binary classes, a simple neural network with no hidden layers and no cycles between units is equivalent to logistic regression if the output layer is a sigmoid (logistic) function of the input layer.
-Similarly, for continuous outcomes, linear regression can be seen as a simple neural network.
-Thus, in some ways, supervised deep learning approaches can be seen as a generalization of regression models that allow for greater flexibility.
+Similarly, for continuous outcomes, linear regression can be seen as a single-layer neural network.
+Thus, in some ways, supervised deep learning approaches can be seen as a extension of regression models that allow for greater flexibility.
```

Comment:
`an extension`

Reply:
maybe as an add-on to that sentence: `and are especially well-suited for modeling non-linear relationships among the input features.`

Reply:
👍